accessibility/README.md

```diff
@@ -108,6 +108,7 @@ We maintain guides to specific areas of accessibility. You may be interested in:
 * [creating accessible emails](03-accessibility-in-emails.md)
 * [creating accessible PDFs](04-accessibility-in-pdfs.md)
 * [learning about effective colour contrast](05-effective-colour-contrast.md)
+* [make good use of the section element](08-guide-to-the-section-element.md)
```

Comment:
after reading the discussion on this, perhaps the chapter should be about _considered use of landmark elements_ instead?